docs/docs/20-concepts/02-mcp-gateway/03-obot-registry.md

```diff
@@ -0,0 +1,298 @@
+# Obot's Open Source Registry
+Each Obot instance connects to our open source registry by default. You can follow this guide to add your MCP server to [Obot’s registry](https://github.com/obot-platform/mcp-catalog). You’ll either submit **just a catalog entry**, or **a catalog entry + a repackaged image** (for stdio servers).
+
+---
+
+## Step 0 — Pick your path
+
+**Question:** What type of MCP server are you shipping?
+
+- **Remote / Streamable HTTP**
+    
+    You expose an HTTP MCP endpoint (hosted) **or** you already have a Docker image that serves the streamable HTTP MCP endpoint.
+    
+    → You only need [**mcp-catalog**](https://github.com/obot-platform/mcp-catalog).
+    
+- **Stdio**
+    
+    You have a stdio server
+    
+    → You’ll touch **both** repos: first [**mcp-images**](https://github.com/obot-platform/mcp-images) (to package it), then [**mcp-catalog**](https://github.com/obot-platform/mcp-catalog).
```

Comment:
We are repackaging existing docker images as well, so this guide isn't quite accurate. Even if they already have a docker image we still want it added to the repackaging list.

If they have a docker image and its stdio only, that's a whole different issue. as far as i know, we dont have a standard way of dealing with that yet.